cypress/integration/admin_portal/tests/coupon_test_template.js

```diff
@@ -0,0 +1,35 @@
+import EnterpriseCoupons from '../helpers/enterprise_coupons'
+
+describe('Login tests', function () {
+  const coupons = new EnterpriseCoupons()
+
+  before(function () {
+    let couponId
```

Comment:
again, why we need to declare this variable? Can not we directly do?
```
this.couponId = response.body.coupon_id
```

What if we declare the `couponId ` variable at `describe` level and then user it like
```
couponId = response.body.coupon_id
```